src/com/scottyplunkett/server/HTTPServer.java

```diff
@@ -3,13 +3,12 @@
 import java.io.*;
 import java.net.ServerSocket;
 import java.net.Socket;
-import java.nio.file.Path;
 import java.util.ArrayList;
 
 public class HTTPServer {
 
     public static void main() throws IOException {
-        main(new String[]{"4444"});
+        main(new String[]{"5000"});
```

Comment:
Magic number.. maybe assign this to a variable so that it is clear what it represents?